Sources/Infrastructure/StreamConsumer.swift

```diff
@@ -69,12 +69,21 @@ class StreamConsumer<StreamType: ColoredStream>: DiagnosticConsumer {
             let lineStr = "\(loc.line)"
             let indentation = "\(indent(lineStr.characters.count))"
             stream.write(" \(indentation)|\n", with: [.cyan])
+            if let prior = file?.lines[loc.line - 2] {
```

Comment:
Use the `safe:` label in the subscript to handle files where the error is on line 0